alma/tests/Unit/Helper/CustomerHelperTest.php

```diff
@@ -24,19 +24,104 @@
 
 namespace Alma\PrestaShop\Tests\Unit\Helper;
 
+use Alma\PrestaShop\Builders\CustomerHelperBuilder;
+use Alma\PrestaShop\Factories\ContextFactory;
+use Alma\PrestaShop\Helpers\OrderHelper;
+use Alma\PrestaShop\Helpers\ValidateHelper;
 use PHPUnit\Framework\TestCase;
 
 class CustomerHelperTest extends TestCase
 {
     public function testGetCustomer()
```

Comment:
Maybe we can split this test with different name for a better understanding of the test and a smaller function

Reply:
I will comment the tests in the function

Reply:
Done